pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -80,8 +80,8 @@ const (
 	maxTargetPoolCreateInstances = 200
 
 	// HTTP Load Balancer parameters
-	// Configure 2 second period for external health checks.
-	gceHcCheckIntervalSeconds = int64(2)
+	// Configure 5 second period for external health checks.
+	gceHcCheckIntervalSeconds = int64(5)
```

Comment:
Let's increase this to 8 s.

Then we have 3 * 8 = 24 s to notice a VM is down.